airflow/www/templates/airflow/trigger.html

```diff
@@ -35,15 +35,20 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
     <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
     <input type="hidden" name="dag_id" value="{{ dag_id }}">
     <input type="hidden" name="origin" value="{{ origin }}">
-  <div class="form-group">
+    <div class="form-group">
       <label class="sr-only" for="execution_date">Logical date</label>
       <div class="input-group">
         {{ form.execution_date(class_="form-control", disabled=False) }}
       </div>
     </div>
-
+    <div class="form-group row">
+      <div class="col-md-2">
+        <label for="run_id">Run id (optional)</label>
```

Comment:
```suggestion
        <label for="run_id">Run id (Optional)</label>
```

Nitpick. But let's be consistent with the Configuration form group.

Reply:
sure